### PR TITLE
CHORE: switch to new requirements

### PR DESCRIPTION
**Pull request by spectacularfailure**
# Description

Switch to new requirements

## Checklist

- [ ] My code follows the style guidelines of this project (`scripts/lint.sh` has no errors)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I added a code examples to illustrate the changes

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -53,7 +68,7 @@ jobs:
           key: ${{ runner.os }}-python-${{ env.pythonLocation }}-${{ hashFiles('pyproject.toml') }}-test-v03
       - name: Install Dependencies
         if: steps.cache.outputs.cache-hit != 'true'
-        run: pip install -r requirements.test.txt .[async-rabbit,async-kafka]
+        run: pip install -r requirements.test.txt .[async-rabbit,async-kafka,async-nats,async-redis,doc]
```
Comment (Lancetnik) [inline .github/workflows/tests.yml]: sqs

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,20 @@
 -e .
+-e .[doc]
 -e .[async-rabbit]
+-e .[async-nats]
 -e .[async-sqs]
+-e .[async-kafka]
+-e .[async-redis]
+
+types-redis
```
Comment (Lancetnik) [inline requirements.txt:9]: there are a `lint` dependencies

File: .github/workflows/tests.yml (hunk 1 above)
Comment (spectacularfailure) [reply .github/workflows/tests.yml]: yes added in recent commit

File: requirements.txt (hunk 2 above)
Comment (spectacularfailure) [reply requirements.txt]: done

File: requirements.txt [hunk 3]
```diff
@@ -6,9 +6,6 @@
 -e .[async-kafka]
 -e .[async-redis]
 
-types-redis
-types-PyYAML
-
 mypy==1.4.1
```
Comment (Lancetnik) [inline requirements.txt:9]: and their too) please, don't rush

**Pull request merged**